apps/src/templates/studioHomepages/Sections.jsx

```diff
@@ -15,11 +16,24 @@ const Sections = React.createClass({
   },
 
   getInitialState() {
-    return {sections: this.props.sections};
+    return {
+      sections: this.props.sections,
```

Comment:
:expressionless: Why did I allow this?  [Props in initial state is an antipattern](https://medium.com/@justintulk/react-anti-patterns-props-in-initial-state-28687846cc2e).  Trying to reconcile this in a merge... Might end up moving this stuff around, FYI.